apex/normalization/fused_layer_norm.py

```diff
@@ -195,6 +297,99 @@ def extra_repr(self):
         return "{normalized_shape}, eps={eps}, " "elementwise_affine={elementwise_affine}".format(**self.__dict__)
 
 
+class FusedRMSNorm(torch.nn.Module):
+    r"""Applies RMS Normalization over a mini-batch of inputs
+
+    Currently only runs on cuda() tensors.
+
+    .. math::
+        y = \frac{x - \mathrm{E}[x]}{ \sqrt{\mathrm{Var}[x] + \epsilon}} * \gamma + \beta
+
+    The mean and standard-deviation are calculated separately over the last
+    certain number dimensions which have to be of the shape specified by
+    :attr:`normalized_shape`.
+    :math:`\gamma` and :math:`\beta` are learnable affine transform parameters of
+    :attr:`normalized_shape` if :attr:`elementwise_affine` is ``True``.
```

Comment:
this looks like a copy-n-paste error - as this version has no bias and no mean subtraction

I think the note below needs updating as well.

Reply:
Could you check the follow-up https://github.com/NVIDIA/apex/pull/1285?

Reply:
oh! I missed that one - thank you! 

looks much better now - with only one small issue - commented in the other PR.